### DISCUSSION TITLE
Add GitHub Sponsor Button

### DISCUSSION THREAD

**Discussion by GovindKrishnan** [AmruthPillai/Reactive-Resume#707] [Ideas]
You might consider adding a GitHub Sponsor Button :heart_decoration: to this repo.

**[GitHub Sponsor Button - Docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository)**

You definitely deserve a coffee :coffee: for your active development. :blush: @AmruthPillai

**Comment by amruthpillai** [comment]
You can actually donate through the button on the website, or this link: https://paypal.me/RajaRajanA

**Reply by amruthpillai** [reply]
And thank you for the support, it is really much much appreciated

**Comment by GovindKrishnan** [comment]
I mean you can still add that button connecting your PayPal just for GitHub :octocat: to know you are accepting tips. Will help GitHub show you Sponsor Button next to Repo Preview in Explore page & Search Results. It's in the repo settings (FUNDING.yml)

![sponsorships-checkbox.png](https://user-images.githubusercontent.com/74248475/158502282-8adaa6a4-caa0-47bd-a85c-2b2a59f30d8d.png)

**Reply by amruthpillai** [reply]
You got it 👍🏻 

![Screenshot 2022-03-16 at 6 10 12 AM](https://user-images.githubusercontent.com/1134738/158520871-f76b64bf-d2b7-4aff-8027-93b6dca17633.png)

**Reply by GovindKrishnan** [reply]
LGTM! 🥂

![Screenshot_2022-03-16-10-54-26-45_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://user-images.githubusercontent.com/74248475/158522346-e4b85aba-9fd1-4cde-ab2f-54b8e8008eeb.jpg)
